.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: php
```

Comment:
I'll enable Travis and Pullapprove after we figure out if we want to move ahead with this.